src/Avalonia.Themes.Fluent/ScrollViewer.xaml

```diff
@@ -66,4 +68,64 @@
     <Setter Property="Opacity" Value="1" />
   </Style>
 
+  <Style Selector="ScrollViewer.menuscroller">
+    <Setter Property="Template">
+      <ControlTemplate>
+        <DockPanel>
+          <RepeatButton DockPanel.Dock="Top"
+                        BorderThickness="0"
+                        Background="Transparent"
+                        HorizontalAlignment="Stretch"
+                        HorizontalContentAlignment="Center"
+                        RenderTransform="{x:Null}"
+                        Command="{Binding LineUp, RelativeSource={RelativeSource TemplatedParent}}">
+            <RepeatButton.IsVisible>
+              <MultiBinding Converter="{x:Static converters:MenuScrollingVisibilityConverter.Instance}"
+                            ConverterParameter="0">
+                <Binding RelativeSource="{RelativeSource TemplatedParent}" Path="VerticalScrollBarVisibility"/>
+                <Binding RelativeSource="{RelativeSource TemplatedParent}" Path="Offset.Y"/>
+                <Binding RelativeSource="{RelativeSource TemplatedParent}" Path="Extent.Height"/>
+                <Binding RelativeSource="{RelativeSource TemplatedParent}" Path="Viewport.Height"/>
+              </MultiBinding>
+            </RepeatButton.IsVisible>
+            <Path Data="M 0 4 L 8 4 L 4 0 Z"/>
```

Comment:
I think it will be better to use same icons, as for menu chevron, but rotated by 90 degrees. For other controls (NumberUpDown and ComboBox we use the same).
![image](https://user-images.githubusercontent.com/3163374/94062582-df44f380-fdb4-11ea-8596-93c6940988e3.png)

Reply:
But I am looking to update this chevron size for menu items (make it a bit smaller), so I can replace this icon in another PR as well.

Reply:
See https://github.com/AvaloniaUI/Avalonia/pull/4750

Reply:
merged.